dev-libs/spsdeclib/spsdeclib-5.1-r1.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Could you also bump to `EAPI="7"`? Thanks!

Reply:
Done.